flight/Modules/Attitude/naze32/attitude.c

```diff
@@ -0,0 +1,730 @@
+/**
```

Comment:
I assume this is in a separate file to have a lightweight attitude estimation for the Naze32. I think it would make sense to have this in a more general file, so it could be used by other targets (e.g. CC3D).